Iguana2024/sprint3
# IGD-52-Create-Dockerfile-for-Flask-app-and-docker-compose.yml

## What type of PR is this? (check all applicable)


## Description
Created Dockerfile for Flask application as well as docker-compose.yml which includes: nginx(as reverse-proxy), app(flak app) and db(mongodb).

## QA Instructions, Screenshots, Recordings
After cloning the repo use 'docker-compose up -d', after containers are up check connectivity to db by typing 'localhost/db' in your browser.

## [optional] Are there any post deployment tasks we need to perform?

We need to use Redis as the buffer between db.
